utils/idmloader/IDM.md

```diff
@@ -0,0 +1,173 @@
+end type GwtDspParamFoundType`
+```
+
+## IDM roadmap
+Phase | Scope | Action | Status
+--- | --- | --- | ---
+1.0 | idm | tool to generate input definitions from dfn files | complete
+1.0 | idm | support generic reading of static mf6 input | complete
+1.0 | idm | evaluate and prototype support for time-varying inputs | ongoing
+1.0 | model subcomponents | update packages requiring static loader | ongoing
+1.1 | idm | support for dynamic loading | not started
+1.1 | model subcomponents | update packages requiring dynamic loader | not started
+1.2 | model subcomponents | remove blockparser from all packages | not started
+1.2 | doc | plan and prioritize required documentation | not started
+1.2 | test | plan and prioritize testing | not started
+2.0 | idm | preparation for 2nd supported input source | future
+2.1 | idm | reader/writer support for 2nd supported input source | future
+
+## Package Update Status
+FTYPE | Status | Comment
+--- | --- | ---
+DIS6 | complete |
+DISV6 | complete |
+DISU6 | complete |
+IC6 | candidate | stage next set
+OC6 | IDM time-varying support needed |
+NPF6 | complete |
+STO6 | candidate | simplified period block, good candidate for early tv support
+CSUB6 | IDM time-varying support needed |
+BUY6 | candidate | in progress
+HFB6 | candidate | period block but no timeseries data
+CHD6 | IDM time-varying support needed | BndType
+WEL6 | IDM time-varying support needed | BndType
+DRN6 | IDM time-varying support needed | BndType
+RIV6 | IDM time-varying support needed | BndType
+GHB6 | IDM time-varying support needed | BndType
+RCH6 | IDM time-varying support needed | BndType
+EVT6 | IDM time-varying support needed | BndType
+MAW6 | IDM time-varying support needed | BndType
+SFR6 | IDM time-varying support needed | BndType
+LAK6 | IDM time-varying support needed | BndType
+UZF6 | IDM time-varying support needed | BndType
+MVR6 | IDM time-varying support needed |
+GNC6 | candidate | stage next set
+OBS6 | candidate | stage next set
+FMI6 | candidate | in progress
+ADV6 | candidate | stage next set
+DSP6 | complete |
+SSM6 | candidate | stage next set
+MST6 | candidate | stage next set
+IST6 | IDM time-varying support needed | BndType
+CNC6 | IDM time-varying support needed | BndType
+SRC6 | IDM time-varying support needed | BndType
+LKT6 | IDM time-varying support needed | GwtAptType
+SFT6 | IDM time-varying support needed | GwtAptType
+MWT6 | IDM time-varying support needed | GwtAptType
+UZT6 | IDM time-varying support needed | GwtAptType
+MVT6 | candidate | stage next set
+API6 | IDM time-varying support needed | BndType
+
+## Package Update process
+###	Update [dfn2f90.py](scripts/dfn2f90.py)
+Add the package dfn file path to the `gwf_dfns` or `gwt_dfns` list structure
+### Run the dfn2f90.py script
+```shell
+cd utils/idmloader/scripts
+python dfn2f90.py
+```
+This will create the new Idm f90 definition file in the appropriate directory, either `src/Model/GroundWaterFlow` or `src/Model/GroundWaterTransport`
```

Comment:
```suggestion
This will create the new IDM Fortran definition file (with a .f90 suffix) in the appropriate directory, either `src/Model/GroundWaterFlow` or `src/Model/GroundWaterTransport`
```